mmdet/models/roi_heads/mask_heads/fcn_mask_head.py

```diff
@@ -62,6 +63,7 @@ def __init__(self,
         self.predictor_cfg = predictor_cfg
         self.fp16_enabled = False
         self.loss_mask = build_loss(loss_mask)
+        self.rescale_mask_to_input_shape = rescale_mask_to_input_shape
```

Comment:
Adding this new arg to `init` of model may be a lack of consideration because it only is used in `onnx_export`,  We may add it to test config

Reply:
Done, take a look at the current solution.